src/BatteryStateOfChargeService.cpp

```diff
@@ -1,13 +1,17 @@
 #include "BatteryStateOfChargeService.h"
+#include <stdio.h>
 
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
+    double ampHourTotal;
+    double ampHourUsed;
```

Comment:
Just to clarify, the namespace isn't really a part of the class (as a static field or otherwise), it's something that can be used in that `.cpp` file.